docker/mesh.config.ts

```diff
@@ -0,0 +1,7 @@
+import { defineConfig as defineServeConfig } from '@graphql-mesh/serve-cli';
+
+export const serveConfig = defineServeConfig({
+  proxy: {
+    endpoint: 'https://countries.trevorblades.com',
```

Comment:
This file is part of the image? why?

Reply:
this is a default config file just so it can be able to run and have a health and readiness endpoints. you can override it with a volume

Reply:
that's a good point

Reply:
Let's remove the default file, it's better to fail it than serving dummy data.

Reply:
sure